tailcfg/tailcfg.go

```diff
@@ -1534,6 +1534,9 @@ type SetDNSRequest struct {
 	Value string
 }
 
+// SetDNSResponse is the repsonse to a SetDNSRequest.
```

Comment:
typo. But you could also send this separately if you wanted.

Reply:
Added a separate commit, don't think we need a separate PR.